### ISSUE TITLE
Feature Request: Markdown support element

### ISSUE THREAD

**Issue by ComanSic** [excalidraw/excalidraw#7564]
I would like to use Excaclidraw for many different purposes and display all kinds of information in it. 
For the text elements, however, I am missing some functionalities that I would need to work effectively with it. For example :
- Lists 
- Links within the text
- Auto wrap when the size of the container changes, and not resize the text
- checklists 
- etc. 

Markdown would already provide many functionalities. Would you consider adding a new element that supports Markdown?

**Comment by niwla23** [comment]
Mabe something like in obsidian canvas, an embedded markdown note.

**Comment by drewgonzales360** [comment]
A very [similar feature](https://github.com/zsviczian/obsidian-excalidraw-plugin#embed-complete-markdown-files-into-your-drawings) exists on the Excalidraw Plugin for Obsidian. You can add markdown elements instead of the regular text and it'll come with all the normal features that markdown has. It's a little unintuitive to use, but this plugin is basically being maintained by one hero.

<img width="1724" alt="Image" src="https://github.com/user-attachments/assets/ccb0ecb7-b112-46ae-8e83-7f6f6c74c244" />

I'd really like this feature to be native in Excalidraw. I'm afraid of features diverging between the Excalidraw I know as an Obsidian plugin and the native Excalidraw.

**Comment by Ni-zav** [comment]
Yes I keep forgetting I have Excalidraw on Obsidian and using the web version, then thought of this implementation. hopefully there is option to enable textbox to have markdown elements
